llama.cpp

```diff
@@ -8027,14 +8033,14 @@ void llama_sample_top_p(struct llama_context * ctx, llama_token_data_array * can
 }
 
 void llama_sample_min_p(struct llama_context * ctx, llama_token_data_array * candidates, float p, size_t min_keep) {
-    if (p <= 0.0f || !candidates->size) {
+    if (p <= 0.0f) {
         return;
     }
+    
+    const int64_t t_start_sample_us = ggml_time_us();
 
     llama_sample_softmax(ctx, candidates);
 
```

Comment:
This will count the `llama_sample_softmax()` time 2 times. Either change to `llama_sample_softmax(nullptr, candidates);` or keep it before the `ggml_time_us()` call